.circleci/config.yml

```diff
@@ -24,11 +69,10 @@ jobs:
            path: plz-out/results
        - run:
            name: Package
-           command: ./plz-out/bin/src/please build //package:all //tools/misc:gen_release //docs:tarball -p
+           command: ./plz-out/bin/src/please build //package:signer //tools/misc:gen_release //docs:tarball -p
```

Comment:
I assume this is intentional and we only need the signer for some reason.

Reply:
Yeah the tarballs etc used to get built by this job; they are now built by the alpine one instead.